app/Http/Controllers/Admin/AdminController.php

```diff
@@ -1,11 +1,35 @@
 <?php
+/**
+ * Parses and verifies the doc comments for files!
+ *
+ * PHP version 7
+ *
+ * @category PHP
+ * @package  PHP_CodeSniffer
+ * @author   LoiTQ <loitq@lifull-tech.vn>
+ * @license  http://matrix.squiz.net/developer/tools/php_cs/licence BSD Licence
+ * @link     http://pear.php.net/package/PHP_CodeSniffer
```

Comment:
Why a product that have no relate to phpcs have comment of phpcs ?

Reply:
Fixed, it deleted